force_gromacs/data_sources/simulation/simulation_model.py

```diff
@@ -0,0 +1,84 @@
+from traits.api import Unicode, Bool, Int, File, Instance
+from traitsui.api import View, Item
+
+from force_bdss.api import (
+    BaseDataSourceModel, BaseDriverEvent, VerifierError
+)
+
+
+class SimulationDataSourceModel(BaseDataSourceModel):
+    """Class that inputs all required parameters for a single
+    Gromacs simulation"""
+
+    #: Name of simulation
+    name = Unicode('simulation')
+
+    #: Number of Molecules
+    n_molecules = Int(1,
+                      desc='Number of constituent molecules',
```

Comment:
Maybe change this to 'molecule types', so as not to confuse with `size` attribute?